### PR TITLE
community/ucpp: upgrade to 1.3.5

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

**Comment by maxice8** [comment]
@tanertas

**Comment by tanertas** [comment]
> @tanertas
@fabled

File: community/ucpp/APKBUILD [hunk 1]
```diff
@@ -1,30 +1,27 @@
 # Contributor: Timo Teräs <timo.teras@iki.fi>
 # Maintainer: Timo Teräs <timo.teras@iki.fi>
 pkgname=ucpp
-pkgver=1.3.2
+pkgver=1.3.5
 pkgrel=0
 pkgdesc="C preprocessor which is embeddable, quick, light and C99 compliant"
-url="https://code.google.com/p/ucpp/"
+options="!check" # No testsuite
+url="https://gitlab.com/scarabeusiv/ucpp"
```
Comment (fabled) [inline community/ucpp/APKBUILD]: This seems to be a fork. Is the other upstream dead? Or is there some specific reasons to switch to this fork?

Comment (maxice8) [reply community/ucpp/APKBUILD]: that upstream had no new releases since 2011.

**Pull request closed without merging**